packages/demo-nextjs/src/app/api/items/latest/route.ts

```diff
@@ -0,0 +1,30 @@
+import { getItems } from "@/lib/db/item/getItems";
+import DBConnect from "@/lib/db/mongoose/connect";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function GET(req: NextRequest) {
+  try {
+    const limit = req.nextUrl.searchParams.get("limit");
+
+    await DBConnect();
+
+    const { items } = await getItems({}, { sort: { createdAt: -1 }, limit: parseInt(limit ?? "5"), skip: 0 });
```

Comment:
need to chage the interface of getItems, need to ad `sortBy: "date" | "name"` and `sort: "desc" | "asc"`